### DISCUSSION TITLE
Investigate using @swc/helpers

### DISCUSSION THREAD

**Discussion by timneutkens** [vercel/next.js#64643] [Ideas]
Revisit https://github.com/vercel/next.js/pull/30205

**Comment by filipesmedeiros** [comment]
this can probably be closed?

**Comment by timneutkens** [comment]
No?

**Comment by filipesmedeiros** [comment]
yes, i'm dumb and got dates mixed together, sorry!

**Comment by SukkaW** [comment]
@timneutkens

Don't know how the issue is going on, but maybe we should add an `experimental` option for it.

Currently, I have 29 `if("Arguments"===c||/^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(c))` in the emitted bundles. Enabling `externalHelpers` definitely helps.

<img width="397" alt="image" src="https://user-images.githubusercontent.com/40715044/157669049-c725ef07-9689-4e21-95f8-306d7418556d.png">
